app/src/main/java/com/framgia/moviedb/screen/home/MovieGroupAdapter.java

```diff
@@ -0,0 +1,67 @@
+package com.framgia.moviedb.screen.home;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import com.framgia.moviedb.R;
+import com.framgia.moviedb.data.model.GroupMovie;
+import java.util.List;
+
+public class MovieGroupAdapter
+        extends RecyclerView.Adapter<MovieGroupAdapter.ViewHolder> {
+    private List<GroupMovie> mList;
+    private final Context mContext;
+
+    public MovieGroupAdapter(List<GroupMovie> list, Context context) {
```

Comment:
n Android, it is common to define methods that take a Context. If you write a method, Context should be the first parameter. And callback always should be the last parameter.